db/migrate/20230622052933_create_users.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :email
+      t.string :phone
+      t.integer :role
+
+      t.timestamps
+    end
```

Comment:
What if I put the values `blank` or `nil`, What will happen? Is there any validations?

Reply:
Bhai, `blank` and `nill` values won't get accepted. Yes, there are validations in the user model.